.golangci.yml

```diff
@@ -0,0 +1,6 @@
+linters:
```

Comment:
remove this file

Reply:
we're adding this to the build checks, so it should be configured: https://github.com/openshift/release/pull/28459